scripts/install.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+set -e
+
+# Determine the platform using 'OS' and 'ARCH'
+initPlatform() {
+    OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+    ARCH=$(uname -m)
+    if [ $OS == "darwin" ] && [ $ARCH == "arm64" ]; then
+        PLATFORM="darwin_arm64"
+    elif [ $OS == "darwin" ] && [ $ARCH == "x86_64" ]; then
+        PLATFORM="darwin_amd64"
+    elif [ $OS == "linux" ] && [ $ARCH == "x86_64" ]; then
+        PLATFORM="linux_amd64"
+    else
+        printf "Platform not supported: os=$OS arch=$ARCH\n"
+        exit 1
+    fi
+}
+
+# Verify that a binary is available for the current combination of 'PLATFORM' and 'VERSION'
+verifyAvailability() {
+    if [ $VERSION == nightly ] && [ $PLATFORM == macos-arm64 ]; then
+        printf "\nNightly builds are currently not published for ${PLATFORM}.\n\n"
+        read -p "Do you want to install the nightly macos-x64 build which can run using Rosetta 2 instead? [y/N]" -n 1 -r -s < /dev/tty
+        printf "\n\n"
+        if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+            exit 0
+        fi
+        PLATFORM=macos-x64
+    fi
+}
```

Comment:
This should no longer be a problem since the new Go CLI can compile for arm from the CI environment